tests/json/expected/pipeline.json

```diff
@@ -1,6 +1,7 @@
 {
-    "id": "PIPELINE_pipeline_0e1c5dd3-9896-4221-bfc8-6924acd11147",
+    "id": "PIPELINE_pipeline_0e1c5dd3-9896-4221-bfc8-6924pjg11147",
```

Comment:
You changed the pipeline_id. It does not match the parent_ids in task.json. Is it on purpose ?

Reply:
ah yes, it's on purpose. I should have changed the parent_ids in the task.json instead, but the meaning is the same. No special reason for it though, simply because I noticed the owner_id and entity_id also don't match up. I'm not sure why but I just decided to follow that as well

Reply:
great.